lib/resolvers/training-pil.js

```diff
@@ -23,7 +23,18 @@ module.exports = ({ models }) => async ({ action, data, id }, transaction) => {
       profile = await Profile.query(transaction).insert(pick(data, 'firstName', 'lastName', 'dob', 'email')).returning('id');
     }
 
-    return TrainingPil.query(transaction).insert({ profileId: profile.id, trainingCourseId: data.trainingCourseId, trainingNeed: data.trainingNeed }).returning('*');
+    return TrainingPil.query(transaction).insert(
```

Comment:
Kiran what will happen if the insert statement fail ?

Reply:
The user will see a "Something went wrong" error page and we'll get an error log in opensearch.

That behaviour isn't changing and it doesn't seem like it's within the scope of this work?

Reply:
this PR only adds new columns to existing insert but FYI worker.js has error handling and all the logic to answer your question